src/Accelerators/NNPA/Compiler/NNPACompilerUtils.cpp

```diff
@@ -72,8 +72,10 @@ void addONNXToZHighPasses(
   pm.addPass(onnx_mlir::createShapeInferencePass());
   pm.addNestedPass<FuncOp>(onnx_mlir::createConstPropONNXToONNXPass());
   // Add instrumentation for Onnx Ops in the same way as onnx-mlir.
-  if (instrumentZHighOps == "" || instrumentZHighOps == "NONE")
-    pm.addNestedPass<FuncOp>(onnx_mlir::createInstrumentONNXPass());
+  if (instrumentZHighOps == "" || instrumentZHighOps == "NONE") {
```

Comment:
nit: braces {} not required  (https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements).

We aren't consistent but we should follow the LLVM coding style because we document we do :-)